climada/hazard/tc_tracks.py

```diff
@@ -1383,7 +1383,7 @@ def write_hdf5(self, file_name, complevel=5):
         df_attrs = pd.DataFrame([t.attrs for t in data], index=ds_combined["storm"].to_series())
         ds_combined = xr.merge([ds_combined, df_attrs.to_xarray()])
 
-        encoding = {v: dict(zlib=True, complevel=complevel) for v in ds_combined.data_vars}
+        encoding = {v: dict(zlib=_zlib_from_DataArray(ds_combined[v]), complevel=complevel) for v in ds_combined.data_vars}
```

Comment:
Line too long (also, rename the function)
```suggestion
        encoding = {
            v: dict(zlib=_zlib_from_dataarray(ds_combined[v]), complevel=complevel)
            for v in ds_combined.data_vars
        }
```